### DISCUSSION TITLE
Newbie needing help

### DISCUSSION THREAD

**Discussion by cmaclennan** [Budibase/budibase#9006] [Help]
Hey Guys, 

I have just stumbled on budibase and think it can do what I'm after or at least I hope it can and am also hoping someone might be able to give me a helping hand or nudge in the right direction.

I am trying to create a system for testing hotel pools/spas I currently have tables setup for the rooms, bookings and various test forms required for room types but now is where i'm hitting some stumbling blocks as I want to create a list of all the rooms that need to be tested when the room is occupied. So if I check in the morning and update any last min bookings I can then generate a list of all the rooms currently occupied on the day and click each as I go down the list to fill in the form with the test results which are all saved for reference.

Any help is greatly appreciated, TIA.

C

**Comment by melohagan** [comment]
Hey @cmaclennan 

This should be achievable through some sort of filtering. I'd need to see more of how your data is structured and some screenshots of what you have so far before I can really offer any help.
